src/settings/ActionProfiles/ViewActionProfile.js

```diff
@@ -175,6 +148,32 @@ export class ViewActionProfile extends Component {
       </AppIcon>
     );
 
+    return (
+      <PaneHeader
+        {...renderProps}
+        id={paneId}
+        paneTitle={paneTitle}
+        paneSub={<FormattedMessage id="ui-data-import.actionProfileName" />}
+        actionMenu={this.renderActionMenu}
+        dismissible
+        onClose={onClose}
+      />
+    );
+  };
+
+  render() {
+    const { tagsEnabled } = this.props;
+
```

Comment:
Please remove a line here and add after the state [here](https://github.com/folio-org/ui-data-import/pull/434/files#diff-63e6fc715be2fbb76352e74f6e40fc4cR341). Usually variables destructured from state and props are grouped and then they are followed with a black line.

Reply:
Done!